### PR TITLE
L4.9.123 2.3.0 0201

### PR DESCRIPTION
**Pull request by thochstein**

### CODE REVIEW

File: recipes-graphics/imx-gpu-apitrace/imx-gpu-apitrace_7.1.0.bb [hunk 1]
```diff
@@ -34,4 +34,4 @@ FILES_${PN} = "${bindir} ${libdir}"
 FILES_${PN}-dbg += "${libdir}/*/*/.debug"
 
 PACKAGE_ARCH = "${MACHINE_SOCARCH}"
-COMPATIBLE_MACHINE = "(mx6q|mx6dl|mx6sx|mx6sl|mx7ulp|mx8mq)"
+COMPATIBLE_MACHINE = "(mx6q|mx6dl|mx6sx|mx6sl|mx7ulp|mx8)"
```
Comment (otavio) [inline recipes-graphics/imx-gpu-apitrace/imx-gpu-apitrace_7.1.0.bb]: This should be another commit ... also, I'd prefer to not use generic overrides if possible ...

**Comment by thochstein** [comment]
This is ready.
